ciongke/github/links.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package github
+
+import "strings"
+
+// Parse Link headers, returning a map from Rel to URL.
```

Comment:
Maybe reference https://tools.ietf.org/html/rfc5988#section-5, and note how much or how little of it you're parsing. If you want to parse it fully, vendor a library. Otherwise, document how limited the parser is.